ntfnhandlers.go

```diff
@@ -225,8 +240,56 @@ func makeNodeNtfnHandlers(cfg *config) (*rpcclient.NotificationHandlers, *collec
 		// memory pool.  It will only be invoked if a preceding call to
 		// NotifyNewTransactions with the verbose flag set to false has been
 		// made to register for the notification and the function is non-nil.
-		OnTxAccepted: func(hash *chainhash.Hash, amount dcrutil.Amount) {
-			// Just send the tx hash and let the goroutine handle everything.
+		// OnTxAccepted: func(hash *chainhash.Hash, amount dcrutil.Amount) {
+		// 	// Just send the tx hash and let the goroutine handle everything.
+		// 	select {
+		// 	case ntfnChans.newTxChan <- &mempool.NewTx{
+		// 		Hash: hash,
+		// 		T:    time.Now(),
+		// 	}:
+		// 	default:
+		// 		log.Warn("newTxChan buffer full!")
+		// 	}
+		// 	//log.Trace("Transaction accepted to mempool: ", hash, amount)
+		// },
+
+		// OnTxAcceptedVerbose is invoked same as OnTxAccepted but is used here
+		// for the explorer mempool monitor to avoid an extra call to dcrd for
+		// the tx details
+		OnTxAcceptedVerbose: func(txDetails *dcrjson.TxRawResult) {
+			msgTx := txhelpers.MsgTxFromHex(txDetails.Hex)
```

Comment:
In general, we want to do as little as possible in these On* handlers since while they are being executed, the RPC client is blocked.  This is why I try to send the bare minimum on a channel to the actual handlers downstream, with a hefty buffer.

So sending the msgTx might be a good idea, then leave the function calls (IsSSGen, SSGenVoteChoices, etc.) to whatever function reads the channel.

Reply:
If we're still seeing issues with hanging and backed up buffered channels, we should look carefully at these handlers.